guidelines.md

```diff
@@ -219,6 +220,19 @@ There are different techniques that can be used such as **Daki Exercise**, **Sto
 - What was bad
 - What can improve
 
+### Deployment
+
+It is proven that a high deploy frequency is a good DevOps practice and gives a lot of benefits in the Agile process. Google considers multiple deploys per day as a [factor to consider a team as an Elite Performer](https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling.).
```

Comment:
I would maybe rephrase this first sentence:

"It is proven that a high deploy frequency is a good DevOps practice and improves productivity of Agile teams."